src/ui/components/trinary_input_string.c

```diff
@@ -161,7 +167,7 @@ static void _render(component_t* component)
         uint8_t string_y = STRING_POS_Y;
 
         if (i == data->string_index &&
-            (confirm_gesture_active || show_title || blink ||
+            (confirm_gesture_active || (!data->title_on_top && show_title) || blink ||
```

Comment:
I'm a bit confused. `show_title` includes `data->title_on_top || ...` so this expr results in:

```
!data->title_on_top && (data->title_on_top || ...rest)
```

which is the same as

```
!data->title_on_top && ...rest
```

Reply:
I was aware, but should I extract `rest` and name it `show_title_if_not_on_top`? :sweat_smile:

Reply:
Actually I did sth else: remove the `data->title_on_top` from `show_title`.